index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="body-wrapper">
+      <!-- nav section -->
+      <div class="mobile-nav">
+        <h1>Welcome</h1>
+        <img src="./assets/main-images/bars.png" alt="" />
+      </div>
+      <div class="hero-wrapper">
+        <nav class="desktop-nav">
+          <ul>
+            <li>
+              <a href="/">Porfolio</a>
+            </li>
+            <li>
+              <a href="/">About</a>
+            </li>
+            <li>
+              <a href="/">Contact</a>
+            </li>
+          </ul>
+        </nav>
```

Comment:
- [ ] You have done very great work to ensure that the portfolio has a header 👏 👍 . However, we need to always use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) tags. This helps us to increase the [accessibility](https://moritzgiessmann.de/accessibility-cheatsheet/) of the portfolio to the screen readers. Therefore I suggest that you ensure that the header is wrapped in a `header` tag ☺️ . 
- [ ] In addition, we should try as much as possible to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=M9ocktxSetpjI0hw-0). I suggest that the `font-family` is changed to match the one of Figma. Also ensure that the `font-size`, and `color` of the portifolio also match one of the Figma.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220284999-b56c2c1e-140e-4195-b794-0ca18114d498.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220285085-34e4edc0-8507-4004-a82f-ba61756ce327.png)